ros_buildfarm/templates/snippet/add_buildfarm_user.Dockerfile.em

```diff
@@ -0,0 +1,3 @@
+# Add user 'buildfarm', removing any existing user with that UID
+RUN if [ $(id -nu @(uid)) ]; then userdel -r $(id -nu @(uid)); fi
+RUN useradd -u @(uid) -l -m buildfarm
```

Comment:
```suggestion
RUN useradd -u @(uid) --non-unique -l -m buildfarm
```
should work as well

Reply:
I'd like to investigate using this mechanism to deal with the collision in the future, but let's save it for a new PR. The delete-then-create approach is what we've been testing with so far.